docs/docs/getting-started/installation/aws.md

```diff
@@ -49,6 +49,12 @@ import Script from "@site/src/components/Script";
 
 If you want to customize your `Airy Core` instance please see our [Configuration Section](configuration.md).
 
+After the installation, you can additionally interact with the components of `Airy Core` with the [kubectl](https://kubernetes.io/docs/tasks/tools/) command line utility, for example:
+
+```sh
+kubectl --kubeconfig ~/.airy/kube.conf get pods
+```
```

Comment:
aws has their way of interacting with a eks cluster (via aws eks update-config and so on) so either we point them to the docs or show here an example of that.

Reply:
What the `aws eks update-config` command does, we do that with the code currently - we fetch the kubeconfig file and we write it in `~/.airy/kube.conf`. The configuration needs to be on the local machine, because otherwise `airy config apply` doesn't work.